ansible/roles/test/tasks/vlan_configure.yml

```diff
@@ -0,0 +1,89 @@
+- fail: msg="Please set ptf_host variable"
+  when: ptf_host is not defined
+
+- fail: msg="Invalid testbed_type value '{{testbed_type}}'"
+  when: testbed_type not in [ 't0' ]
+
+- debug: var=minigraph_portchannels
+
+- debug: var=minigraph_port_indices
+
+- name: Generate VLAN ports information
+  template: src=roles/test/templates/vlan_info.j2
+            dest=/tmp/vlan_info.yml
+  connection: local
+
+- name: Load VLAN ports info from file
+  include_vars: '/tmp/vlan_info.yml'
+
+- debug: var=vlan_ports_list
+- debug: var=vlan_intf_list
+
+- name: Flush all IP addresses on the LAGs
+  shell: ip addr flush {{ item.attachto }}
+  with_items:
+    - "{{ minigraph_portchannel_interfaces }}"
+  become: true
+
+- name: Delete all IP addresses on the LAGs in config DB
```

Comment:
This is a workaround since there is no command line to configure interface IP and write to configDB now.  Is there any plan to support this?

Reply:
We plan to support IP and Lag configuration though configDb in next release March.

---
In reply to: [155185549](https://github.com/Azure/sonic-mgmt/pull/375#discussion_r155185549) [](ancestors = 155185549)